.rubocop.yml

```diff
@@ -95,3 +95,7 @@ Lint/SuppressedException:
 # Enforce LF line endings, even when on Windows
 Layout/EndOfLine:
   EnforcedStyle: lf
+
+Lint/AmbiguousOperator:
```

Comment:
Can we just fix this by using parens?

background: I think optional method parens are one of ruby's worst design decisions.

Reply:
I don't think so? I'm not sure how rubocop expects the splat to be called, but this is what I get using parens:
```
be rubocop lib/bolt/apply_target.rb
Inspecting 1 file
W

Offenses:

lib/bolt/apply_target.rb:9:17: W: Lint/AmbiguousOperator: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the * if it should be a multiplication.
    attr_reader *(ATTRIBUTES)
                ^
lib/bolt/apply_target.rb:9:18: C: Style/RedundantParentheses: Don't use parentheses around a constant.
    attr_reader *(ATTRIBUTES)
                 ^^^^^^^^^^^^
lib/bolt/apply_target.rb:10:19: W: Lint/AmbiguousOperator: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the * if it should be a multiplication.
    attr_accessor *COMPUTED
```

This will at least only disable it for that file. I can see if there's a recommended syntax, or try turning the constants into class methods or something?

Reply:
```rb
    attr_reader(*ATTRIBUTES)
    attr_accessor(*COMPUTED)
```

Reply:
https://github.com/rubocop-hq/rubocop/issues/2000